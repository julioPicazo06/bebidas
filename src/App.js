import { Fragment } from "react";
import Formulario from "./components/Formulario";
import Header from "./components/Header";
import { CategoriasProvider } from "./Context/CategoriaContext";

const App = () => {
  return (
    <CategoriasProvider>
      <Header/>
      <div className="container mt-5">
        <div className="">
            <Formulario/>
        </div>
      </div>
    </CategoriasProvider>
  );
}

export default App;
