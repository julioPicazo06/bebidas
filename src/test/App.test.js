import { shallowToJson } from "enzyme-to-json";
import App from "../App";

describe('Coinside con el snapp', () => {
    test('prueba en <App></App>', () => {
        const wrapper = shallowToJson(<App/>);
        expect(wrapper).toMatchSnapshot();
    });
});