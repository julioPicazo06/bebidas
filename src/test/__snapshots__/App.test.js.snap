// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Coinside con el snapp prueba en <App></App> 1`] = `null`;
